src/nunit.analyzers.tests/ConstActualValueUsage/ConstActualValueUsageAnalyzerTests.cs

```diff
@@ -0,0 +1,138 @@
+using Gu.Roslyn.Asserts;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Analyzers.ConstActualValueUsage;
+using NUnit.Framework;
+
+namespace NUnit.Analyzers.Tests.ActualConstValueUsage
+{
+    public class ActualConstValueUsageAnalyzerTests
+    {
+        private readonly DiagnosticAnalyzer analyzer = new ActualConstValueUsageAnalyzer();
```

Comment:
Why not static? Also one new line after is enough.

Reply:
fixed